### ISSUE TITLE
Feature: Accounts In-App Modal

### ISSUE THREAD

**Issue by ardeay** [zesty-io/manager-ui#491]

**Comment by ardeay** [comment]
@kakoga please comment your thoughts on this. Start from our Feb 8th 2021 meetings

**Comment by kakoga** [comment]
When accessing accounts from Manager UI it's very disruptive to the flow that accounts UI loads in the window that you're in. When a user is in the middle of working on something (have unsaved work on an item) and happen to click the Accounts link - there is no option to save that work before Accounts UI loads. (I'll enter a ticket for this)  

Because of that - it would be really helpful and a better user experience - I think - if Accounts could open in a modal or in a new browser tab to avoid disrupting workflow.

**Comment by d88naimi** [comment]
@giseleblair Is this still an issue or can this be closed?  Can you provide the steps to reproduce this functionality please?

**Comment by giseleblair** [comment]
@d88naimi yes this is still an issue, here is the workflow: https://www.loom.com/share/4daa2e03d7de45409e5695ee33403a67
